examples/paho-mqtt/Makefile

```
@@ -16,7 +16,17 @@ QUIET ?= 1
 
 USEPKG += paho-mqtt
 
+CFLAGS += -DESP_WIFI_SSID=\"define yours\" -DESP_WIFI_PASS=\"secret\"
+
+ifeq ($(BOARD),esp32-wrover-kit)
```

This is relevant for all ESP32 boads. So you should use
```suggestion
ifneq (,$(filter arch_esp32,$(FEATURES_USED)))
```